pubnub-kotlin/pubnub-kotlin-impl/src/integrationTest/kotlin/com/pubnub/api/integration/SubscribeIntegrationTests.kt

```diff
@@ -1365,6 +1365,138 @@ class SubscribeIntegrationTests : BaseIntegrationTest() {
         }
     }
 
+    @Test
+    fun whenSubscribingToAlreadySubscribedChannelShouldNotResubscribeButShouldEmitSubscriptionChangedStatus() {
```

Comment:
It's a pretty long name. What about `shouldEmitSubscriptionChangedWhenAlreadySubscribed` or something similar?